docs/commands/alias.md

```diff
@@ -1,11 +1,17 @@
 # alias
-This command allows you to define shortcuts for other common commands.
+This command allows you to define shortcuts for other common commands. By default, they only apply to the current session. To persist them, add `--save`.
+
+Syntax: `alias {flags} <name> [<parameters] {<body>}`
```

Comment:
Looks like you're missing a `>` after parameters

Reply:
Good catch, good fix